### ISSUE TITLE
environment variable ZS_LoadModulePath appears to not be handled by bootstrap.sh

### ISSUE THREAD

**Issue by ShaunDwyer** [zabbix/zabbix-community-docker#55]
Hi,

It appears that ZS_LoadModulePath isn't handled at all by /config/init/boostrap.sh as appears to be the case with other environment variables.

I'm using monitoringartist/zabbix-3.0-xxl:latest.

Cheers!
-Shaun

**Comment by jangaraj** [comment]
Fix available in `monitoringartist/zabbix-3.0-xxl:dev` image for now:

```
[root@dockerhost ~]# docker run -d --name zabbix --env='ZS_LoadModulePath=/tmp' monitoringartist/zabbix-3.0-xxl:dev
4f2d83533ac301787b02abc25b97df3c3f57934a937d62115a6f22947b165154
[root@dockerhost ~]# docker exec -ti zabbix grep LoadModulePath /usr/local/etc/zabbix_server.conf
LoadModulePath=/tmp
```

**Comment by jangaraj** [comment]
Done. Fixed images are available.
